spec/services/search_identities_spec.rb

```diff
@@ -206,8 +206,8 @@
     end
 
     context 'cro_number search' do
-      context 'when query matches' do
```

Comment:
❓ By adding the *wildcard* option, you're no longer able to do precise matches on CRO number? Is the only search option for this field supposed to be with *wildcards*?